ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/assisted/ovn/hypershift-agent/provision/cucushift-installer-rehearse-baremetalds-assisted-ovn-hypershift-agent-provision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-baremetalds-assisted-ovn-hypershift-agent-provision
```

Comment:
I did not see a dedicated setting for `ovn`, so I guess we should rename the folder path to `cucushift/installer/rehearse/baremetalds/assisted/hypershift-agent/provision` ?

Reply:
```
workflow:
  as: cucushift-installer-rehearse-baremetalds-assisted-ovn
  steps:
    allow_best_effort_post_steps: true
    allow_skip_on_success: true
    pre:
    - chain: cucushift-installer-rehearse-baremetalds-assisted-provision
    test:
    - ref: cucushift-installer-check
    post:
    - chain: cucushift-installer-rehearse-baremetalds-assisted-deprovision
    env:
      DEVSCRIPTS_CONFIG: |
        IP_STACK=v4
        NETWORK_TYPE=OVNKubernetes
        EXTRA_WORKER_VCPU=8
        EXTRA_WORKER_MEMORY=16384
        EXTRA_WORKER_DISK=100
        PROVISIONING_NETWORK_PROFILE=Disabled
        REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
```
In our workflow, the mgmt cluster is OVN, so I have added the name of this chain.

Reply:
I noticed the workflow explicitly set the OVN network type, but not in provision/deprovision chain, if you want to place the chains in ovn folder, I suggest to move the `env` to provision chain.

Reply:
or only move provsion/deprovision chain to `cucushift/installer/rehearse/baremetalds/assisted/hypershift-agent/` folder, leave the workflow yaml file in the ovn folder.

Reply:
ok, I moved the workflow into hypershift dir to make it special for hypershift now.